<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TabReactions</class>
 <widget class="QWidget" name="TabReactions">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1422</width>
    <height>821</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Reactions</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QSplitter" name="splitter_5">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="childrenCollapsible">
      <bool>false</bool>
     </property>
     <widget class="QWidget" name="verticalLayoutWidget_4">
      <layout class="QGridLayout" name="gridLayout_6">
       <item row="2" column="0">
        <widget class="QPushButton" name="btnAddReaction">
         <property name="text">
          <string>Add</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QPushButton" name="btnRemoveReaction">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Remove</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0" colspan="2">
        <widget class="QTreeWidget" name="listReactions">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <attribute name="headerVisible">
          <bool>false</bool>
         </attribute>
         <column>
          <property name="text">
           <string notr="true">1</string>
          </property>
         </column>
        </widget>
       </item>
       <item row="0" column="0" colspan="2">
        <widget class="QLabel" name="lblReactions">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="font">
          <font>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Reactions:</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QGroupBox" name="grpReactions">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
        <horstretch>1</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="title">
       <string>Reaction Settings</string>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <layout class="QGridLayout" name="gridLayout_9" rowstretch="1,0,0,0,0,0,0">
         <property name="horizontalSpacing">
          <number>6</number>
         </property>
         <item row="1" column="1" colspan="3">
          <widget class="QComboBox" name="cmbReactionLocation"/>
         </item>
         <item row="4" column="1">
          <widget class="QPushButton" name="btnAddReactionParam">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>Add parameter</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1" colspan="3">
          <widget class="QLineEdit" name="txtReactionName"/>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="lblReactionParamsLabel">
           <property name="text">
            <string>Params:</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item row="5" column="0">
          <widget class="QLabel" name="lblReactionRateLabel">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>Rate:</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="lblStoichiometryLabel">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>Species:</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item row="4" column="2">
          <widget class="QPushButton" name="btnRemoveReactionParam">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>Remove parameter</string>
           </property>
          </widget>
         </item>
         <item row="6" column="1">
          <widget class="QPushButton" name="btnSaveReactionChanges">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>Save changes</string>
           </property>
          </widget>
         </item>
         <item row="4" column="3">
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="5" column="1" colspan="3">
          <widget class="QPlainTextMathEdit" name="txtReactionRate">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="tabChangesFocus">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="2" column="1" colspan="3">
          <widget class="QTreeWidget" name="listReactionSpecies">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="rootIsDecorated">
            <bool>true</bool>
           </property>
           <column>
            <property name="text">
             <string>Species</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Stoichiometry</string>
            </property>
           </column>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="lblReactionNameLabel">
           <property name="text">
            <string>Name:</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item row="6" column="2" colspan="2">
          <widget class="QLabel" name="lblReactionRateStatus">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="palette">
            <palette>
             <active>
              <colorrole role="WindowText">
               <brush brushstyle="SolidPattern">
                <color alpha="255">
                 <red>255</red>
                 <green>0</green>
                 <blue>0</blue>
                </color>
               </brush>
              </colorrole>
              <colorrole role="Text">
               <brush brushstyle="SolidPattern">
                <color alpha="255">
                 <red>255</red>
                 <green>0</green>
                 <blue>0</blue>
                </color>
               </brush>
              </colorrole>
              <colorrole role="ButtonText">
               <brush brushstyle="SolidPattern">
                <color alpha="255">
                 <red>255</red>
                 <green>0</green>
                 <blue>0</blue>
                </color>
               </brush>
              </colorrole>
              <colorrole role="PlaceholderText">
               <brush brushstyle="SolidPattern">
                <color alpha="128">
                 <red>255</red>
                 <green>0</green>
                 <blue>0</blue>
                </color>
               </brush>
              </colorrole>
             </active>
             <inactive>
              <colorrole role="WindowText">
               <brush brushstyle="SolidPattern">
                <color alpha="255">
                 <red>255</red>
                 <green>0</green>
                 <blue>0</blue>
                </color>
               </brush>
              </colorrole>
              <colorrole role="Text">
               <brush brushstyle="SolidPattern">
                <color alpha="255">
                 <red>255</red>
                 <green>0</green>
                 <blue>0</blue>
                </color>
               </brush>
              </colorrole>
              <colorrole role="ButtonText">
               <brush brushstyle="SolidPattern">
                <color alpha="255">
                 <red>255</red>
                 <green>0</green>
                 <blue>0</blue>
                </color>
               </brush>
              </colorrole>
              <colorrole role="PlaceholderText">
               <brush brushstyle="SolidPattern">
                <color alpha="128">
                 <red>255</red>
                 <green>0</green>
                 <blue>0</blue>
                </color>
               </brush>
              </colorrole>
             </inactive>
             <disabled>
              <colorrole role="WindowText">
               <brush brushstyle="SolidPattern">
                <color alpha="255">
                 <red>160</red>
                 <green>162</green>
                 <blue>162</blue>
                </color>
               </brush>
              </colorrole>
              <colorrole role="Text">
               <brush brushstyle="SolidPattern">
                <color alpha="255">
                 <red>168</red>
                 <green>169</green>
                 <blue>169</blue>
                </color>
               </brush>
              </colorrole>
              <colorrole role="ButtonText">
               <brush brushstyle="SolidPattern">
                <color alpha="255">
                 <red>160</red>
                 <green>162</green>
                 <blue>162</blue>
                </color>
               </brush>
              </colorrole>
              <colorrole role="PlaceholderText">
               <brush brushstyle="SolidPattern">
                <color alpha="128">
                 <red>35</red>
                 <green>38</green>
                 <blue>39</blue>
                </color>
               </brush>
              </colorrole>
             </disabled>
            </palette>
           </property>
           <property name="font">
            <font>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="lblReactionLocationLabel">
           <property name="text">
            <string>Location:</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item row="3" column="1" colspan="3">
          <widget class="QTableWidget" name="listReactionParams">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="cornerButtonEnabled">
            <bool>true</bool>
           </property>
           <attribute name="horizontalHeaderStretchLastSection">
            <bool>true</bool>
           </attribute>
           <attribute name="verticalHeaderVisible">
            <bool>false</bool>
           </attribute>
           <column>
            <property name="text">
             <string>Name</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Value</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Units</string>
            </property>
           </column>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QPlainTextMathEdit</class>
   <extends>QPlainTextEdit</extends>
   <header>qplaintextmathedit.hpp</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
